src/main/webapp/app/course/manage/course-update.component.ts

```diff
@@ -258,12 +259,10 @@ export class CourseUpdateComponent implements OnInit {
      * @function set course icon
      * @param event {object} Event object which contains the uploaded file
      */
-    setCourseImage(event: any): void {
-        this.imageChangedEvent = event;
-        if (event.target.files.length) {
-            const fileList: FileList = event.target.files;
-            this.courseImageFile = fileList[0];
-            this.courseImageFileName = this.courseImageFile.name;
+    setCourseImage(event: Event): void {
+        const element = event.currentTarget as HTMLInputElement;
+        if (element.files && element.files[0]) {
```

Comment:
```suggestion
        if (element.files?.[0]) {
```

Reply:
Weird, I actually had the same idea, but then it showed me a syntax error that confused me. Now it works 🤷